_Sidebar.md

```diff
@@ -42,6 +42,7 @@ Guides
   * [Mod functions](https://github.com/Steamodded/smods/wiki/Mod-functions)
   * [UI Structure](https://github.com/Steamodded/smods/wiki/UI-Guide)
   * [Utility Functions](https://github.com/Steamodded/smods/wiki/Utility)
+  * [All About G](https://github.com/Steamodded/smods/wiki/G)
```

Comment:
Not quite sure I like this name (it's not immediately obvious this is a variable)

Reply:
I'll wrap it in backticks, then.